attiny/attiny.ino

```diff
@@ -170,6 +183,12 @@ void testBlink(int x) {
   }
 }
 
+void setBlinks(int i) {
+  if (blinks <= 0) {
+    blinks = i;
+  }
```

Comment:
I guess it's ok that if 2 blink events happen close together the second one is lost?

Reply:
Yeah, we could buffer the blinks.. But this at least prevents getting wrong messages.